packages/clay-css/src/scss/variables/_tables.scss

```diff
@@ -324,8 +324,8 @@ $c-table: map-deep-merge(
 		),
 		autofit-col: (
 			justify-content: center,
-			padding-left: nth($table-cell-padding, 2),
-			padding-right: nth($table-cell-padding, 2),
+			padding-left: nth($table-cell-padding, 1),
```

Comment:
@ethib137 , this will add `0.5 rem`, so I removed the `calc()` function

Reply:
@ilzamcmed this is referencing the top and bottom padding of a table cell. I don't think we should do this.

Reply:
I also found use cases in Portal that are using utilities to put 0.25rem padding left and right on autofit-col's in tables. @ethib137 would 0.25rem be acceptable? I think 4px/8px spacing is what Lexicon generally uses

Reply:
Thank you, @pat270! Evan clarified the function of `nth`, as I had misunderstood it. I will wait for Evan's feedback regarding your suggestion and will send the necessary fix afterward.

Reply:
Hey @marcoscv, what are your thoughts on Patrick's comment above?

Reply:
Hey guys, sorry for the delay, marcoscv is my personal user and I don’t get notifications.
From my side any way to fix it is ok, you guys are the experts! 🎉